gui/slick/js/new/config_backuprestore.js

```diff
@@ -1,5 +1,119 @@
-$(document).load(function(){
+$(document).ready(function(){
     $('#backupDir').fileBrowser({ title: 'Select backup folder to save to', key: 'backupPath' });
     $('#backupFile').fileBrowser({ title: 'Select backup files to restore', key: 'backupFile', includeFiles: 1 });
```

Comment:
@OmgImAlexis I see that but still not working. If you click "restore" first after load the page, it shows the restore title. But if then you click backup, it uses the restore message. The other way around too if you refresh page. Don't know how to fix it. Look like browser is saving window settings and not re-creating the window